apis/gateway/v1beta2/apirule_types.go

```diff
@@ -223,6 +223,5 @@ type CorsPolicy struct {
 	AllowOrigins     StringMatch `json:"allowOrigins,omitempty"`
 	AllowCredentials *bool       `json:"allowCredentials,omitempty"`
 	ExposeHeaders    []string    `json:"exposeHeaders,omitempty"`
-	// +kubebuilder:validation:Format=duration
-	MaxAge *metav1.Duration `json:"maxAge,omitempty"`
+	MaxAge           uint64      `json:"maxAge,omitempty"`
```

Comment:
I think adding this validation will provide a better error message in case of an invalid input, e.g. negative number.
``` 
The APIRule "httpbin-rule" is invalid: spec.corsPolicy.maxAge: Invalid value: -4: spec.corsPolicy.maxAge in body should be greater than or equal to 0
``` 
instead of

```
Error from server: error when creating "STDIN": conversion webhook for gateway.kyma-project.io/v1beta2, Kind=APIRule failed: json: cannot unmarshal number -4 into Go struct field CorsPolicy.spec.corsPolicy.maxAge of type uint64
```
```suggestion
        // +kubebuilder:validation:Minimum=0
	MaxAge           uint64      `json:"maxAge,omitempty"`
```